Sources/Private/CoreAnimation/ValueProviderStore.swift

```diff
@@ -93,14 +94,13 @@ final class ValueProviderStore {
   // MARK: Private
 
   private let logger: LottieLogger
-
   private var valueProviders = [(keypath: AnimationKeypath, valueProvider: AnyValueProvider)]()
 
-  /// Retrieves the most-recently-registered Value Provider that matches the given keypat
+  /// Retrieves the most-recently-registered Value Provider that matches the given keypath.
   private func valueProvider(for keypath: AnimationKeypath) -> AnyValueProvider? {
     // Find the last keypath matching the given keypath,
     // so we return the value provider that was registered most-recently
-    valueProviders.last(where: { registeredKeypath, _ in
+    valueProviders.reversed().first(where: { registeredKeypath, _ in
```

Comment:
is `.reversed().first` different from `.last`?

Reply:
No different, thanks! Reverted to using `last `.